docs/content/policy-reference.md

```diff
@@ -724,6 +726,14 @@ all_paths[entity_name]
 
 {{< builtin-table cat=graphql title="GraphQL" >}}
 
+{{< info >}}
+Custom [GraphQL `@directive`][graphql-spec-directives] definitions defined by your GraphQL framework will need to be included manually as part of your GraphQL schema string in order for validation to work correctly on GraphQL queries using those directives.
+A common way to add these directive definitions if they are defined separately from your schema is to `concat` them to your schema string right before use.
+Directives defined as part of the GraphQL specification (`@skip`, `@include`, `@deprecated`, and `@specifiedBy`) are supported by default, and do not need to be added to your schema manually.
+{{< /info >}}
+
+   [graphql-spec-directives]: http://spec.graphql.org/October2021/#sec-Language.Directives
```

Comment:
[nit] A little uncommon, this reference style. But it doesn't hurt, I suppose.

Reply:
For the `concat` bit, would a separate example block be worthwhile, you think?

Reply:
Also, the reference style doesn't seem to be supported in `{{< info >}}` blocks, so I'm going to try the direct-link style, and if that fails, just paste the raw link in there. :sweat_smile:

Reply:
Direct-link (`[text](link)`) style seems to be supported. :+1: